build.sbt

```diff
@@ -19,6 +19,10 @@ lazy val `client-lib` = project
   .dependsOn(common)
   .settings(
     name := "client-lib",
+    libraryDependencies ++= Seq(
```

Comment:
If we add any more dependencies it'd probably make sense to pull these into a val so the structure of this file remains consistent?